app/javascript/react/screens/App/Mappings/screens/MappingWizard/components/MappingWizardGeneralStep/MappingWizardGeneralStep.js

```diff
@@ -22,6 +22,8 @@ class MappingWizardGeneralStep extends React.Component {
         targetProvider: determineTargetProvider(editingMapping)
       });
     }
+
+    fetchConversionHostsAction(fetchConversionHostsUrl);
```

Comment:
I'm wondering if, now that we're not clearing the conversion hosts from the store, we should only fetch conversion hosts if the array is empty.

The only downside that I can think of is stale data.  The user would have to refresh their browser in case a conversion host is somehow added w/o them navigating away from this page

Reply:
Ok, good call.